src/main/java/uk/gov/dvsa/PdfGenerator.java

```diff
@@ -46,6 +46,10 @@ public PdfGenerator() {
     public ApiGatewayResponse handleRequest(Map<String, Object> input, Context context) {
         long start = System.nanoTime();
         eventLogger.logEvent(EventType.CERT_REQUEST_RECEIVED);
+
```

Comment:
should this log be removed? is it just here for testing?

Reply:
Yes can you remove this please. I only added it as part of 9866 to test the cert-gen logic.